modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -697,7 +803,7 @@ DROP DATABASE customers IF EXISTS
 The `DROP DATABASE` command will remove a database entirely.
 
 [role=enterprise-edition not-on-aura]
-=== Dump and delete databases
+== Dump and delete databases
```

Comment:
I did that because having just one subsection looks a bit odd and also contains the possibility to dump your database, so it's a different task, not the same task, but performed differently.

Reply:
it still uses the `DROP DATABASE` command though? To me if feels odd that the same command have 2 sections for it's variations (it's just that DESTROY DATA is the default and don't need to be specified) instead of just one...

If you want more than one subsection we could make the `IF EXISTS` part a subsection as well

Reply:
You're right. I added them as subsections.